src/DebuggerData.cpp

```diff
@@ -285,7 +285,7 @@ int Breakpoints::breakpointCount()
 
 bool Breakpoints::isBreakpoint(quint16 addr, QString* id, bool checkSlot)
 {
-	if (int index; (index = findBreakpoint(addr) != -1)) {
+	if (int index; (index = findBreakpoint(addr)) != -1) {
```

Comment:
Please write this as:
```if (int index = findBreakpoint(addr); index != -1) {```
Ah, but it's the same line as I mentioned in the review of the previous commit.

Please try to avoid making fixes like this in multiple steps in your commit series. This change belonged in the previous commit.